lib/ansible/plugins/connection/psrp.py

```diff
@@ -94,6 +94,15 @@
     vars:
     - name: ansible_psrp_connection_timeout
     default: 30
+  read_timeout:
```

Comment:
We mustn't have a sanity check for all plugins but we should have `version_added: '2.8'` for this new option.